lib/syntax.js

```diff
@@ -113,7 +113,7 @@ module.exports = {
         }
     },
 
-    isStrictModeReservedWord: function(id) {
+    isStrictModeReservedWord: function(id, ecmaFeatures) {
```

Comment:
You're right of course. I did it mostly for consistency with `isKeyword` and to avoid the boolean parameter code smell.